roles/openshift_logging_elasticsearch/tasks/generate_secret.yaml

```diff
@@ -1,5 +1,22 @@
 ---
 # secret
+- name: Checking for passwd.yml
+  stat:
+    path: "{{ generated_certs_dir }}/passwd.yml"
+    get_checksum: false
+    get_attributes: false
+    get_mime: false
+  register: passwd_file
+  check_mode: no
+
+- when: not passwd_file.stat.exists or openshift_logging_elasticsearch_prometheus_sa not in ( _logging_metrics_proxy_passwd['content'] | b64decode | from_yaml )
```

Comment:
This duplicates code from `openshift_logging_elasticsearch/tasks/main.yaml` - why can't we put the checking in that file?

Reply:
> This duplicates code from `openshift_logging_elasticsearch/tasks/main.yaml` - why can't we put the checking in that file?

This is the 3rd step of this bz1747305:
```
3. redeploy certificates using playbook
ansible-playbook playbooks/openshift-logging/redeploy-certificates.yml
```
The playbook imports https://github.com/openshift/openshift-ansible/blob/release-3.11/playbooks/openshift-logging/private/redeploy-certificates.yml, where I could see a link to `openshift_logging_elasticsearch/tasks/main.yaml`.

Reply:
ah, ok